core/packages/test/scripts/configure-bridgehub.sh

```diff
@@ -17,7 +17,31 @@ config_inbound_queue()
     local pallet="30"
     local callindex="01"
     local payload="0x$pallet$callindex$(address_for OutboundQueue | cut -c3-)"
-    send_governance_transact_from_relaychain $bridgehub_para_id "$payload" 180000000000 900000
+    send_governance_transact_from_relaychain $bridgehub_para_id "$payload"
+}
+
+config_pallet_owner()
+{
+    local owner=$(echo $bridgehub_pallets_owner | cut -c3-)
+    local option="01"
+
+    # config owner of inbound queue
+    local pallet="30"
```

Comment:
Which runtime do these indexes come from? I'm only seeing Rococo, but that looks like the index is 48: https://github.com/Snowfork/cumulus/blob/be6d97cd/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs#L740

Reply:
Yes, `30` is just the hex value of 48.

Reply:
Oh, it's a hex value 🤦 Sorry